[Bug]: Cannot compile for iOS due to error "Undefined symbols for architecture arm64"

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When adding facebook_app_events to our app, we are getting these warnings and errors. 

```
ld: warning: argument missing after -rpath
    ld: warning: Could not find or use auto-linked library 'swiftFileProvider'
    ld: warning: Could not find or use auto-linked library 'swift_Concurrency'
    ld: warning: Could not find or use auto-linked library 'swiftDataDetection'
    ld: warning: Could not find or use auto-linked library 'stdc++'
    ld: warning: Could not find or use auto-linked framework 'AVFAudio'
    ld: warning: Could not find or use auto-linked framework 'DataDetection'
    Undefined symbols for architecture x86_64:
      "_swift_FORCE_LOAD$_swiftDataDetection", referenced from:
          _swift_FORCE_LOAD$swiftDataDetection$_FBAudienceNetwork in FBAudienceNetwork(FBAudienceNetwork.o)
         (maybe you meant: _swift_FORCE_LOAD$swiftDataDetection$_FBAudienceNetwork)
      "OBJC_CLASS$_SKAdImpression", referenced from:
          objc-class-ref in FBAudienceNetwork(FBAdSKAdNetworkManager.m.o)
      "_swift_FORCE_LOAD$_swiftFileProvider", referenced from:
          _swift_FORCE_LOAD$swiftFileProvider$_FBAudienceNetwork in FBAudienceNetwork(FBAudienceNetwork.o)
         (maybe you meant: _swift_FORCE_LOAD$swiftFileProvider$_FBAudienceNetwork)
    ld: symbol(s) not found for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

When we remove facebook_app_events from pubspec.yaml the project compiles just fine. 


### Expected Behavior

Adding the flutter facebook_app_events dependency should compile successfully for iOS.

### Steps To Reproduce

1. Add facebook_app_events: 0.14.5 to the pubspec.yaml
2. Build the project for iOS using XCode.


### Platform

iOS

### Flutter Doctor

```shell
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.5.3, on Mac OS X 10.15.7 19H15 darwin-x64, locale
    en-MT)
[!] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for
      more details.
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.2)
[✓] IntelliJ IDEA Community Edition (version 2020.1.2)
[✓] Connected device (2 available)
```

--- a/ios/Classes/SwiftFacebookAppEventsPlugin.swift
+++ b/ios/Classes/SwiftFacebookAppEventsPlugin.swift
@@ -86,7 +86,7 @@ public class SwiftFacebookAppEventsPlugin: NSObject, FlutterPlugin {
     private func handleLogEvent(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
         let arguments = call.arguments as? [String: Any] ?? [String: Any]()
         let eventName = arguments["name"] as! String
-        let parameters = arguments["parameters"] as? [AppEvents.ParameterName.RawValue: Any] ?? [AppEvents.ParameterName.RawValue: Any]()
+        let parameters = arguments["parameters"] as? [AppEvents.ParameterName: Any] ?? [AppEvents.ParameterName: Any]()
         if arguments["_valueToSum"] != nil && !(arguments["_valueToSum"] is NSNull) {
             let valueToDouble = arguments["_valueToSum"] as! Double
             AppEvents.logEvent(AppEvents.Name(eventName), valueToSum: valueToDouble, parameters: parameters)

--- a/ios/facebook_app_events.podspec
+++ b/ios/facebook_app_events.podspec
@@ -13,9 +13,9 @@ Flutter plugin for Facebook Analytics and App Events
   s.public_header_files = 'Classes/**/*.h'
   s.static_framework = true
   s.dependency 'Flutter'
-  s.dependency 'FBSDKCoreKit', '~> 12.0'
+  s.dependency 'FBSDKCoreKit', '~> 12.1.0'
   s.dependency 'FBAudienceNetwork', '~> 6.9.0'
   s.swift_version       = '5.0'
 
-  s.ios.deployment_target = '9.0'
+  s.ios.deployment_target = '10.0'
 end

